src/components/AppLayout/AppLayout.tsx

```diff
@@ -733,8 +735,11 @@ export class AppLayout extends Component<any, IAppLayoutState> {
                                                     <Avatar
                                                         alt="You"
                                                         src={
-                                                            this.state
-                                                                .currentUser
+                                                            this.props.avatarURL
+                                                                ? this.props
+                                                                      .avatarURL
+                                                                : this.state
+                                                                      .currentUser
                                                                 ? this.state
                                                                       .currentUser
                                                                       .avatar_static
```

Comment:
Could we maybe clean this up a bit with nullish coalescing? Something like the following (though a bit unsure if it'd work):

```suggestion
                                                            this.props.avatarURL
                                                                ?? this.state.currentUser?.avatar_static
                                                                ?? ""
```

Reply:
Does the latest Prettier support nullish coalescing? I know it supports optional chaining, but maybe it doesn’t quite support the coalescing just yet...